x-pack/plugins/cloud_security_posture/server/create_transforms/create_transforms.ts

```diff
@@ -9,11 +9,18 @@ import { TransformPutTransformRequest } from '@elastic/elasticsearch/lib/api/typ
 import type { ElasticsearchClient, Logger } from '@kbn/core/server';
 import { errors } from '@elastic/elasticsearch';
 import { latestFindingsTransform } from './latest_findings_transform';
+import { latestVulnerabilitiesTransform } from './latest_vulnerabilities_transforms';
 
 const LATEST_TRANSFORM_V830 = 'cloud_security_posture.findings_latest-default-0.0.1';
 const LATEST_TRANSFORM_V840 = 'cloud_security_posture.findings_latest-default-8.4.0';
-
-const PREVIOUS_TRANSFORMS = [LATEST_TRANSFORM_V830, LATEST_TRANSFORM_V840];
+const LATEST_TRANSFORM_V880_001 = 'cloud_security_posture.findings_latest-default-0.0.1';
+const LATEST_VULNERABILITIES_V880 = 'cloud_security_posture.vulnerabilities_latest-default-8.8.0';
+const PREVIOUS_TRANSFORMS = [
+  LATEST_TRANSFORM_V830,
```

Comment:
please refactor the naming of `LATEST_TRANSFORM_*` - to better distinguish between findings and vulnerabilities